### DISCUSSION TITLE
ACR1252u "Error: Could not SELECT default selected: 0x6280"

### DISCUSSION THREAD

**Discussion by edent** [martinpaljak/GlobalPlatformPro#348] [Q&A] [answered]
I'm using an ACR1252u (connected via USB-C if that makes a difference).

Trying to read a VivoKey Apex chip. This is running "NXP JCOP 4 P71". It has multiple applets installed - including NDEF and a FIDO2 applet.

If I place the chip on the reader and try to scan it, I get:

```
java -jar gp.jar -dvi -r 1
# gp -dvi -r 1
# GlobalPlatformPro v21.12.31-55-g52c3581
# Running on Linux 6.6.10-76060610-generic amd64, Java 11.0.21 by Ubuntu
[DEBUG] TerminalManager - Processing 2 readers with 1 as preferred and null as ignored
[DEBUG] TerminalManager - Chose ACS ACR1252 1S CL Reader [ACR1252 Dual Reader PICC] 00 00 by index 1
[DEBUG] TerminalManager - Preferred reader: Optional[ACS ACR1252 1S CL Reader [ACR1252 Dual Reader PICC] 00 00]
# SCardConnect("ACS ACR1252 1S CL Reader [ACR1252 Dual Reader PICC] 00 00", T=*) -> T=1, 3B8A80010031C173C8400000900090
A>> T=1 (4+0000) 00A40400 00 
A<< (0011+2) (18ms) 6175746820657863656564 6280
Error: Could not SELECT default selected: 0x6280
# SCardDisconnect("ACS ACR1252 1S CL Reader [ACR1252 Dual Reader PICC] 00 00", false) tx:5/rx:13 in 41ms
```

If I run the command without the chip on the reader, it says `SCARD_E_NO_SMARTCARD` as expected.

I'd like to be able to read the chip, erase unwanted applets, and load new ones. Does anyone know if that's possible?

**Comment by martinpaljak** [comment, marked as answer]
This is a warning from the card manager that it is "locked" - you have exhausted some kind of internal counters. Are you sure you know the keys to the chip?

**Reply by edent** [reply]
Thanks. I had previously locked the NDEF app using the Fidesmo Android app - but that shouldn't have locked the whole thing.

Interestingly, the https://github.com/fidesmo/fdsm/ Java file is able to read the details of the loaded applets - but it can't load or destroy them.  I'll go back to Fidesmo and see what they suggest.

Thanks for the quick response - I was tearing my hair out!
